### ISSUE TITLE
User Activity Fetch

### ISSUE THREAD

**Issue by dadevguy** [Rednexie/discord-id#2]
Can we also get user activity?

**Comment by Rednexie** [comment]
ofc

**Comment by Rednexie** [comment]
ofc but i won't be able to add caching

**Comment by dadevguy** [comment]
yeah also it doesnt fetch all my details like i literally have nitro since ages but it says nitro: false 
and doesnt show nitro badges

**Comment by Rednexie** [comment]
api might have changed, i am coming with a bigger project called everything discord which will probably have this features and more

**Comment by dadevguy** [comment]
good luck!
